.npmrc

```diff
@@ -1,2 +1,3 @@
 strict-peer-dependencies=false
+auto-install-peers=true
```

Comment:
There was a slightly tricky issue where the dependency for `react-query` in `@mysten/core` created a separate module installation in `pnpm-lock.yaml` resulting in two different react-query contexts (one for the wallet/explorer and one for `@mysten/core`).

The fix for this was to make `react`, `react-dom`, and `react-query` peer dependencies in `@mysten/core`, and so I flipped `auto-install-peers` to true so we can actually use those modules inside our core library

https://github.com/TanStack/query/issues/3595#issuecomment-1248074333
https://github.com/pnpm/pnpm/issues/5351

Reply:
Ahhhh I figured out why two separate entries were being created in `pnpm-lock.yaml`, I missed adding `react-dom` as a dependency to `@mysten/core` 🤦🏼‍♂️. I reverted the change to `.npmrc`, so I think we should be good here now

There should be only a single version of `react-query` now:
```
  /@tanstack/react-query/4.22.0_biqbaboplfbrettd7655fr4n2y:
    resolution: {integrity: sha512-P9o+HjG42uB/xHR6dMsJaPhtZydSe4v0xdG5G/cEj1oHZAXelMlm67/rYJNQGKgBamKElKogj+HYGF+NY2yHYg==}
    peerDependencies:
      react: ^16.8.0 || ^17.0.0 || ^18.0.0
      react-dom: ^16.8.0 || ^17.0.0 || ^18.0.0
      react-native: '*'
    peerDependenciesMeta:
      react-dom:
        optional: true
      react-native:
        optional: true
    dependencies:
      '@tanstack/query-core': 4.22.0
      react: 18.2.0
      react-dom: 18.2.0_react@18.2.0
      use-sync-external-store: 1.2.0_react@18.2.0
    dev: false
```